src/Faker/Provider/Base.php

```diff
@@ -168,9 +168,17 @@ public static function randomAscii()
      *
      * @return array New array with $count elements from $array
      */
-    public static function randomElements(array $array = array('a', 'b', 'c'), $count = 1, $allowDuplicates = false)
+    public static function randomElements($array = array('a', 'b', 'c'), $count = 1, $allowDuplicates = false)
```

Comment:
can you read the type hint?

Reply:
i don't know what you mean by this.

Reply:
Just revert your changes to this line

Reply:
the whole point of this PR is to accept things other than arrays, so if I add the type hint back, that negates all of the new code.

Reply:
ok, you're right, my bad